### DISCUSSION TITLE
Github Student denied 19 times. While everything is good

### DISCUSSION THREAD

**Discussion by Syxpi** [community/community#56627] [GitHub Education]
### Hi everyone,

Hello Github, how are you? Me how to tell you that I'm not well, because Github Student starts to annoy me, 19 times that I make my request, yes really 19 TIMES.
Each time refused because I did not complete my Github profile

"You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio."
![chrome_6Ahnro3XfH](https://github.com/community/community/assets/109110286/841b71bc-b1c9-40d7-a8c2-5c2f4373f2e7)

Except it's bullshit, I changed my profile more than 40 times, with my first and last name, putting the address of my high school (because I'm in France)
by putting the name of my high school, by doing EVERYTHING possible, each refused for the same reason. Really it's starting to annoy me.

Just Github if you don't want me to have the Student pack, say it directly, as I don't mind doing it since the fucking December 26, 2022.

Because of you I can't take advantage of 3/4 of the software and extensions I need for my projects.
I literally put everything possible, but no, it openly tells me to get out and not try again, otherwise I'll fail on the spot.

Spoiler: After 30 seconds after it tells me this: 
![image](https://github.com/community/community/assets/109110286/0e3016dc-7997-41f5-a624-a5d6d801bf12)
he actually tells me that my request is rejected, with always and again the same answer.
Just put back the manual checks no? because the automatic checks carried out with your shitty bot, it's quickly annoying

As long as the manual checks are not put back in place, I would not ask again.

Good. I think I've finished saying what I had to say.
PS: all this is google translation, I'm sorry if you don't understand anything x)

**Comment by Orafilynie** [comment]
I can clearly say that's true. Github ask me an academic email to use my school (it is mandatory) but they stopped issuing emails since 2017... They are clearly late to make things works and i can relate to this post that the student is a clear unmanaged sphagetti mess (cmon they just refuse half of people just to still make juicy addicted money from frustated students). Others platform took me 5 minutes and didnt asked me for an abandonned email... Github is very late from others platform.

**Reply by GhostOf0days** [reply]
Do you understand how many people Github supports internationally? It would be very hard to keep track of every single school and changes in their email. Furthermore, schools need to notify Github. Internal changes can't be found easily by an external organization.

**Reply by Logxn** [reply]
It would be hard to keep track of every single school?
Here is a tip: GitHub should follow its own requirements, and accept official documents as an alternative.
My requests were denied 4-5 times by now because they want to force me to use my school mail.
However GitHub only supports one out of two endpoints for my school.

**Comment by Ntshangase** [comment]
Hi @Syxpi 

- After changing your profile information it's always good practice to logout of your GitHub profile then login again, after that you can proceed to submit your application
- make sure the documents you submitted also have an English translation next to them.

**Reply by Syxpi** [reply]
I disconnect and reconnect 5 times after making modifications, to be sure that everything is correctly modified.


And having a translation of my French liaison book, it's going to be complicated :/

**Reply by Ntshangase** [reply]
Maybe try using another document which can be easily translated, as long as it proves that you are a registered student.

**Reply by Syxpi** [reply]
Well, apart from my notebook, I can't do anything else.
because if I screen my transcript (digitally) it will be refused because we can say that it has been modified.
basically I'm stuck there.
Considering the GitHub verification bot is crap.

**Comment by samus-aran** [comment]
Hi @Syxpi 

Let's work through these rejection reasons.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
- Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Let us know how you get on 👍

**Reply by Syxpi** [reply]
Hello, thank you for your response.

My original document is not in English. It doesn't help and instant reject my request.

Next, you mention an English translation. What do you mean by that? Because it doesn't give me clear indications.

Regarding the GitHub details, what do you mean by that? Does it refer to my first name, last name, or something else on my profile?

I don't use any variation of my name or a nickname, it's just my username.

I've tried all possibilities, but nothing works.

I took a photo of my school ID multiple times, but none of them were accepted.
I also took a screenshot of Pronote (a software used in some high schools in France), but that didn't work either.

I ended up giving up.
